### PR TITLE
[incubator/vault] Add Ingress labels

### PR DESCRIPTION
**Pull request by mathieuherbert**
Hi,
please find the following PR
**What this PR does / why we need it**:
- Add custom ingress labels
- fix default replicas count on Readme

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
- the capacity to have custom labels in the ingress 
- error in documentation
**Special notes for your reviewer**:

### CODE REVIEW

**Comment by mathieuherbert** [comment]
CLA signed

File: incubator/vault/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Vault, a tool for managing secrets
 name: vault
-version: 0.8.0
+version: 0.8.1
```
Comment (timstoop) [inline incubator/vault/Chart.yaml]: You're adding a new feature, please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]
Please add the ingress.labels key to both the values.yaml and the README.

**Comment by mathieuherbert** [comment]
@timstoop thanks for the review, I did a commit :)

**Pull request merged**